addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -202,6 +203,65 @@ const PopupWidget = publicWidget.Widget.extend({
 
 publicWidget.registry.popup = PopupWidget;
 
+publicWidget.registry.cookies_bar = PopupWidget.extend({
+    selector: '#website_cookies_bar',
+    start() {
+        if (
+            this.el.querySelector(".o_cookies_popup")
+            && window.location.pathname === new URL(
+                this.el.querySelector(".o_cookies_bar_text_policy").href
+            ).pathname
+        ) {
+            // Stable fix: using raw HTML instead of a template.
+            this.toggleEl = $(`
+            <button class="o_cookies_bar_toggle btn btn-info btn-sm rounded-circle d-flex align-items-center justify-content-center position-absolute">
+                <i class="fa fa-eye" alt="" aria-hidden="true"></i> <span class="o_cookies_bar_toggle_label"></span>
+            </button>
+            `)[0];
+            this._toggleCookiesBar();
+            this._onToggleCookiesBar = this._toggleCookiesBar.bind(this);
+            this.toggleEl.addEventListener("click", this._onToggleCookiesBar);
+        }
+        return this._super(...arguments);
+    },
+    destroy() {
+        if (this.toggleEl) {
+            if (this.el.querySelector(".modal-content").classList.contains("d-none")) {
+                this._toggleCookiesBar();
+            }
+            this.toggleEl.removeEventListener("click", this._onToggleCookiesBar);
+            this.toggleEl.remove();
+        }
+        this._super(...arguments);
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    _toggleCookiesBar() {
```

Comment:
If a `"Backdrop"` is enabled for cookies bar, it will remain visible after hiding the popup and will also block the page scroll :grimacing: we need to remove / restore it too...

Reply:
Good catch! Thanks